superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

```diff
@@ -157,7 +157,7 @@ function DashboardList(props: DashboardListProps) {
       ({ json = {} }) => {
         setDashboards(
           dashboards.map(dashboard => {
-            if (dashboard.id === json.id) {
+            if (dashboard.id === json?.result.id) {
```

Comment:
Sure! But there will be another problem, here I only need to update the title, colorSchema, json_metadata, etc., but the modified time is controlled by `changed_on_delta_humanized`, the response does not have this field.

Reply:
Can we just add that param to the response in backend? @zhaoyongjie @villebro

Reply:
@kgabryje makes sense - I'll see if it's easy to add it.

Reply:
Thanks! BTW,  If we update the dashboard's title, since the list is sorted by modified time, not only update the title, the order of the list also needs to be updated. This may be a case I need to consider. So I think `refreshData` can simplify a lot, but it will add additional requests.

Reply:
I have added `changed_on_delta_humanized` field in the RESTful response.

https://github.com/apache/superset/pull/15542

Reply:
I think changing the order of the list will be surprising for the user. It may look like the dashboard they were editing has disappeared.